collection-loop.php

```diff
@@ -0,0 +1,41 @@
+
+<?php
+function loopCollection(array $result): string
+{
+  $collectedwine = '';
+  for ($i=0; $i < count($result); $i++)
+  {
```

Comment:
Opening brace for `for` and `while` loops should be on same line as `for` or `while`.